modules/eap/setup/eap/modules/module.yaml

```diff
@@ -31,11 +31,11 @@ ports:
 artifacts:
 - name: eap-image-maven-repo
   target: maven-repo.zip
-  md5: 115acd5c7fbcaea84f843c593e850515
-  url: http://$DOWNLOAD_SERVER/released/middleware/eap7/7.4.2/jboss-eap-7.4.2-image-builder-maven-repository.zip
-# CIAM-711: Update RH-SSO server overlay to 'RH-SSO 7.5.0.CR4' build
+  md5: 2c0ab034200cafbffb0b808bf2f59e67
+  url: http://$DOWNLOAD_SERVER/released/middleware/eap7/7.4.3/jboss-eap-7.4.3-image-builder-maven-repository.zip
+# RHSSO-1824: Update RH-SSO server overlay to 'RH-SSO 7.5.2.CR1' build
 - name: keycloak-server-overlay.zip
-  md5: 4150d09f7bf5d524b4173f8850a06617
+  md5: TODO
```

Comment:
Why was this changed to TODO?

Reply:
IIRC, the server zip's hash needs to go there, but the server zip doesn't exist yet. Ideally I'd update this PR before merging

Reply:
Oh right. The zips are uploaded on our internal servers now

Reply:
Updated. I found the hash in the maven repo, since we're apparently not putting the overlay zip in the candidate anymore?

Reply:
I checked in PNC and the hash is correct